__init__.py

```diff
@@ -385,7 +381,7 @@ def handle_automation_intent(self, message):
             self.ha.execute_service("homeassistant", "turn_on",
                                     data=ha_data)
 
-    def handle_sensor_intent(self, message):
+    def handle_sensor(self, message):
```

Comment:
Same comment, should we make this method private?

Reply:
can be - will change

Reply:
done